numpy_challenge.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+if __name__ == "__main__":  # creatint 3 different numpy arrays
+    arr_1 = np.eye(3, dtype=int)
+    arr_2 = np.arange(9).reshape((3, 3))
+    arr_3 = np.fromstring('7 3', dtype=int, sep=' ')  # fav number of Sheldon Cooper
+
+
+def matrix_multiplication(a, b):  # a and b are matrices
+    return np.matmul(a, b)
+
+
+def multiplication_check(a):  # a is an array of matrices
+    for i in range(1, len(a)):
+        if np.shape(a[i])[0] != np.shape(a[i - 1])[1] or np.shape(a[i])[1] != np.shape(a[i - 1])[0]:
```

Comment:
Второе условие лишнее 
![image](https://user-images.githubusercontent.com/48160953/199851174-7db80122-571c-4ab5-9c70-00086bab464d.png)
```suggestion
        if np.shape(a[i])[0] != np.shape(a[i - 1])[1]:
```